test/viewmodels/dashboard_viewmodel_test.dart

```diff
@@ -45,15 +56,38 @@ void main() {
     PreferencesFlag.progressBarCard: 1
   };
 
+  // Session
+  final Session session = Session(
+      shortName: "É2020",
+      name: "Ete 2020",
+      startDate: DateTime(2020).subtract(const Duration(days: 1)),
+      endDate: DateTime(2020).add(const Duration(days: 1)),
+      endDateCourses: DateTime(2022, 1, 10, 1, 1),
+      startDateRegistration: DateTime(2017, 1, 9, 1, 1),
+      deadlineRegistration: DateTime(2017, 1, 10, 1, 1),
+      startDateCancellationWithRefund: DateTime(2017, 1, 10, 1, 1),
+      deadlineCancellationWithRefund: DateTime(2017, 1, 11, 1, 1),
+      deadlineCancellationWithRefundNewStudent: DateTime(2017, 1, 11, 1, 1),
+      startDateCancellationWithoutRefundNewStudent: DateTime(2017, 1, 12, 1, 1),
+      deadlineCancellationWithoutRefundNewStudent: DateTime(2017, 1, 12, 1, 1),
+      deadlineCancellationASEQ: DateTime(2017, 1, 11, 1, 1));
+
   group("DashboardViewModel - ", () {
```

Comment:
Can you add a test for the futureToRun of the session progress?

Reply:
done